internal/btf/core.go

```diff
@@ -423,9 +456,23 @@ func coreCalculateFixup(local Type, localID TypeID, target Type, targetID TypeID
 			if err != nil {
 				return zero, err
 			}
+			return fixupField(uint32(localSize), uint32(targetSize))
 
-			return fixup(uint32(localSize), uint32(targetSize))
+		case reloFieldLShiftU64:
+			targetSize, err := Sizeof(targetField.Type)
+			if err != nil {
+				return zero, err
+			}
+			var target uint32
+			if byteOrder == binary.LittleEndian {
```

Comment:
same computation here as in libbpf: https://elixir.bootlin.com/linux/latest/source/tools/lib/bpf/relo_core.c#L664.